templates/rhpam70-kieserver-postgresql.yaml

```diff
@@ -324,62 +395,80 @@ objects:
             value: "${KIE_SERVER_CONTROLLER_HOST}"
           - name: KIE_SERVER_CONTROLLER_PORT
             value: "${KIE_SERVER_CONTROLLER_PORT}"
+          - name: KIE_SERVER_CONTROLLER_PROTOCOL
+            value: "${KIE_SERVER_CONTROLLER_PROTOCOL}"
+          - name: KIE_SERVER_CONTROLLER_TOKEN
+            value: "${KIE_SERVER_CONTROLLER_TOKEN}"
           - name: KIE_SERVER_ID
             value: "${KIE_SERVER_ID}"
           - name: KIE_SERVER_HOST
             valueFrom:
               fieldRef:
                 fieldPath: status.podIP
-          - name: KIE_SERVER_CONTAINER_DEPLOYMENT
-            value: "${KIE_SERVER_CONTAINER_DEPLOYMENT}"
-          - name: KIE_SERVER_PWD
-            value: "${KIE_SERVER_PWD}"
           - name: KIE_SERVER_USER
             value: "${KIE_SERVER_USER}"
+          - name: KIE_SERVER_PWD
+            value: "${KIE_SERVER_PWD}"
+          - name: KIE_SERVER_CONTAINER_DEPLOYMENT
+            value: "${KIE_SERVER_CONTAINER_DEPLOYMENT}"
           - name: MAVEN_REPO_URL
             value: "${MAVEN_REPO_URL}"
           - name: MAVEN_REPO_USERNAME
             value: "${MAVEN_REPO_USERNAME}"
           - name: MAVEN_REPO_PASSWORD
             value: "${MAVEN_REPO_PASSWORD}"
-          - name: MAVEN_REPO_SERVICE
-            value: "${APPLICATION_NAME}-rhpamcentr"
           - name: MAVEN_REPO_PATH
             value: "/maven2/"
+          - name: KIE_SERVER_ROUTER_SERVICE
+            value: "${KIE_SERVER_ROUTER_SERVICE}"
           - name: KIE_SERVER_PERSISTENCE_DIALECT
-            value: "${KIE_SERVER_PERSISTENCE_DIALECT}"
+            value: "org.hibernate.dialect.PostgreSQLDialect"
           - name: KIE_SERVER_PERSISTENCE_DS
             value: "${KIE_SERVER_PERSISTENCE_DS}"
-          - name: KIE_SERVER_PERSISTENCE_TM
-            value: "${KIE_SERVER_PERSISTENCE_TM}"
-          - name: DB_SERVICE_PREFIX_MAPPING
-            value: "${APPLICATION_NAME}-postgresql=DB"
-          - name: DB_JNDI
-            value: "${DB_JNDI}"
-          - name: DB_USERNAME
-            value: "${DB_USERNAME}"
-          - name: DB_PASSWORD
-            value: "${DB_PASSWORD}"
-          - name: DB_DATABASE
-            value: "${DB_DATABASE}"
-          - name: TX_DATABASE_PREFIX_MAPPING
-            value: "${APPLICATION_NAME}-postgresql=DB"
-          - name: DB_MIN_POOL_SIZE
-            value: "${DB_MIN_POOL_SIZE}"
-          - name: DB_MAX_POOL_SIZE
-            value: "${DB_MAX_POOL_SIZE}"
-          - name: DB_TX_ISOLATION
-            value: "${DB_TX_ISOLATION}"
+          - name: DATASOURCES
+            value: "RHPAM"
+          - name: RHPAM_DATABASE
+            value: "${KIE_SERVER_POSTGRESQL_DB}"
+          - name: RHPAM_JNDI
+            value: "${KIE_SERVER_PERSISTENCE_DS}"
+          - name: RHPAM_DRIVER
+            value: "postgresql"
+          - name: RHPAM_JTA
+            value: "true"
+          - name: RHPAM_TX_ISOLATION
+            value: "TRANSACTION_READ_UNCOMMITTED"
```

Comment:
There is a specific reason to hard code to UNCOMMITED? I'd suggest to make it configurable, at least for the ejb timers the default is TRANSACTION_READ_COMMITTED but configurable through the env EJB_TIMER_TX_ISOLATION.

Reply:
I understand where @spolti is coming from, and if it were me I would probably make it configurable just to align with how we're making similar things configurable as mentioned. However, I also see @bmozaffa 's point about only making things configurable if they highlight  changes we expect users to make...

I'm curious, @bmozaffa, why READ_UNCOMMITTED instead of READ_COMMITTED? It seems like READ_UNCOMMITTED is not a safe option... If it is going to be hardcoded, shouldn't we configure it with a safer isolation level?

Reply:
I actually didn't select a value. It must have been there in whatever baseline template I worked with. Will change to committed.